dubbo-config/dubbo-config-spring/src/main/java/com/alibaba/dubbo/config/spring/extension/SpringExtensionFactory.java

```diff
@@ -47,6 +52,24 @@ public static void removeApplicationContext(ApplicationContext context) {
                 }
             }
         }
+
+        if (logger.isDebugEnabled()) {
+            logger.debug("No spring extension(bean) named:" + name + ", type:" + type.getName() + " found. Will try to get bean by type only.");
+        }
+
+        for (ApplicationContext context : contexts) {
+            try {
+                return context.getBean(type);

```

Comment:
whether the getBean() returns null or not, this line will end the function progress, is it the real purpose?

Reply:
We should consider multiple matching conditions for the same interface type.